ts/components/ui/AlertModal.tsx

```diff
@@ -21,49 +20,41 @@ const styles = StyleSheet.create({
     width: "auto",
     backgroundColor: IOColors.white,
     padding: themeVariables.contentPadding,
-    borderRadius: 8
+    marginHorizontal: IOVisualCostants.appMarginDefault,
+    borderCurve: "continuous",
+    borderRadius: 16
   }
 });
 
-type Props = Readonly<{
+type AlertModalProps = Readonly<{
   message: string;
 }>;
 
 /**
  * A custom alert to show a message
  */
-export class AlertModal extends React.PureComponent<Props> {
-  private subscription: NativeEventSubscription | undefined;
-  constructor(props: Props) {
-    super(props);
-  }
+export const AlertModal = ({ message }: AlertModalProps) => {
+  useEffect(() => {
```

Comment:
What about
`useHardwareBackButton(() => true);`
?

Reply:
@Vangaorth Good catch! Replaced in the following commit → https://github.com/pagopa/io-app/pull/5763/commits/5f25be1ea006efe4e08311b94af7b3b5371a2d0f